tests/test_dynamo_db_base_block.py

```diff
@@ -0,0 +1,30 @@
+from unittest.mock import MagicMock, patch
+from nio.common.signal.base import Signal
+from nio.util.support.block_test_case import NIOBlockTestCase
+from ..dynamo_db_base_block import DynamoDBBase
+
+
+class PassDynamoDB(DynamoDBBase):
+
+    def execute_signals_query(self, tables, signals):
+        return signals
+
+
+@patch(DynamoDBBase.__module__ + '.connect_to_region')
+class TestDynamoDBBase(NIOBlockTestCase):
+
+    @patch('boto.dynamodb2.table.BatchTable.put_item')
+    @patch(DynamoDBBase.__module__ + '.Table.count')
+    def test_save_batch(self, count_func, put_func, connect_func):
```

Comment:
This test might need a new name, we're not really saving. Is the point to just make sure that we split up the signals by table name before operating on them?